application/clicommands/ProcessCommand.php

```diff
@@ -150,15 +160,34 @@ protected function renderProblemTree($tree, $useColors = false, $depth = 0)
             if ($useColors) {
                 $state = $this->screen->colorize($state, $colors[0], $colors[1]);
             }
-
-            $output .= sprintf(
-                "%s%s %s %s\n",
-                str_repeat('  ', $depth),
-                $node instanceof BpNode ? $node->getOperator() : '-',
-                $state,
-                $node->getAlias()
-            );
-            $output .= $this->renderProblemTree($subtree['children'], $useColors, $depth + 1);
+            if ($htmlcolors) {
+                $state = substr($state, 1, strlen($state)-2);
+                $lstate = strtolower($node->getStateName());
+                $config = $this->Config();
+                FakeRequest::setConfiguredBaseUrl($config->get("Main", "baseurl"));
+                $output.="<table class=''><tbody><tr>"
```

Comment:
`<table`**` class=''`**`>`? What for? And why one table per node and not one table for all?